gui/amberWorkflow.py

```diff
@@ -61,11 +65,9 @@ def _initialize_local_model(self, treeView):
     def _initialize_irods_model(self, treeView):
         self.irodsmodel = IrodsModel(treeView)
         treeView.setModel(self.irodsmodel)
-        irodsRootColl = self.context.irods_environment.config.get(
-                'irods_home', '/'+self.context.irods_connector.zone)
-        self.irodsmodel.setHorizontalHeaderLabels([irodsRootColl,
-                                              'Level', 'iRODS ID',
-                                              'parent ID', 'type'])
+        irodsRootColl = self.ienv.get('irods_home', '/'+self.conn.zone)
```

Comment:
Unrelated to this PR, but I notice it due to the line changes, for uniformity either replace with:
`home_coll_str = utils.path.iRODSPath('/', self.conn.zone, 'home')
irods_root_coll = self.ienv.get('irods_home', home_coll_str)`
Or please update the reference in other tabs like example.

Reply:
Indeed, this is a legacy issue.  I can easily go through these and update them now, or they can wait until I update the style of the particular modules in turn.  Perhaps the latter is best to keep this PR as simple as possible.  A compromise might be to update only those modules that are already being changed and leave the rest for later.